demisto_sdk/commands/create_artifacts/tests/contet_creator_test.py

```diff
@@ -0,0 +1,31 @@
+import filecmp
+from demisto_sdk.commands.create_artifacts.content_creator import *
+
+
+def test_copy_dir_files():
```

Comment:
`copy_dir_files` is internal details. 
instead test public functionality like `ContentCreator.run` function. 
And add test description, what is the behaviour that you are testing here

Reply:
It's one path after the `create_content`, and I wanted to avoid the zipping on circle.
I agree we should do it, bur would preferred it in a 2nd batch, as this part have no tests at all.